home/forms/domain_model.py

```diff
@@ -0,0 +1,103 @@
+from typing import NamedTuple
```

Comment:
This information should eventually come from the catalogue. I decided against implementing that now, but I thought it was worth extracting out  into a separate module, as there are a lot of places we use this metadata

Reply:
One benefit of pulling this from the catalogue is we can include counts of the number of assets in each domain, which would assist users browsing the catalogue just to get a sense of what's there.